conanfile.py

```diff
@@ -0,0 +1,47 @@
+
+from conans import ConanFile, CMake, tools
+from conans.tools import os_info, SystemPackageTool
+from pathlib import Path
+import os
+import shutil
+
+
+class uwebsocketConan(ConanFile):
+    name = "usockets"
+    version = "0.0.5"
+    license = "Apache 2.0"
+    author = "kaiyin keezhong@qq.com"
```

Comment:
Did you pull this from your ass or what?

Reply:
I just used a template. Why the fuck should you be so rude?

Reply:
Read the license. Expect a DMCA takedown the moment you publish this nonsense.